src/scripts/seed-db-countries.ts

```diff
@@ -0,0 +1,39 @@
+import path from 'path'
+import fs from 'fs'
+import { dataSource } from '../configs/dbConfig'
+import { Country } from '../entities/country.entity'
+
+// Countries file must be in the same directory as the seeding file in build directory.
+const countriesDataFileName = 'countries.json'
+const countriesFilePath = path.join(__dirname, countriesDataFileName)
+
+export const seedCountries = async (): Promise<void> => {
```

Comment:
As we discussed, let's move this script to the existing seed script @Disura-Randunu

Reply:
sure. will update that

Reply:
@anjula-sack btw, are we seeding the actual database through this script or manually inserting through the shell or something?

Reply:
We will be running the command

Reply:
@anjula-sack updated as disussed